### PR TITLE
adds new v3 dxp configure doc

### PR DESCRIPTION
**Pull request by brentkulwicki**
fixes: https://github.com/FlatFilers/Guides/issues/835

### CODE REVIEW

File: upgrade/v3_dxp_configure.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Upgrade with @flatfile/configure"
+description: "Upgrade to Platform using DXP Configure"
+icon: "triangle-exclamation"
+---
+<Warning>
+    This package simply extends functionality within our Configure package. If you're not already using that package, this is an unnecessary step and you should [start here](/upgrade/v3_upgrade) instead.
```
Comment (ashleygmulligan2) [inline upgrade/v3_dxp_configure.mdx:7]: Can you say @flatfile/configure and link to it on npm

File: upgrade/v3_dxp_configure.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</Warning>
+
+## Before starting
```
Comment (ashleygmulligan2) [inline upgrade/v3_dxp_configure.mdx:10]: Before you begin

File: upgrade/v3_dxp_configure.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Before you get started, there are both some new and redesigned terms to learn. Check out our Portal 3 Upgrade Guide's [before you begin section](/upgrade/v3_upgrade#before-you-begin) to learn about these concepts.
+
+## Using the plugin
+If you're using the Portal 3 class-based syntax for creating your Workbooks, Sheets, etc., the `dxpConfigure` method lets you pass in those classes and create your new Platform setup without having to move away from the classes for now.
```
Comment (ashleygmulligan2) [inline upgrade/v3_dxp_configure.mdx:14]: Link to v3 docs here where we introduce class-based syntax and that package

**Review by ashleygmulligan2 — CHANGES_REQUESTED** [review]
Do you have this in your upgrade example repo? Can you link to it if so?

**Comment by brentkulwicki** [comment]
> Do you have this in your upgrade example repo? Can you link to it if so?

Do not have this in the Upgrade repo at this point, but I will add something to it that tracks along with the configure package and then can update to link to the repo from this guide too

**Review by ashleygmulligan2 — APPROVED** [review]

**Pull request merged**